frontend/components/MainPage/header/LastTransaction.tsx

```diff
@@ -0,0 +1,79 @@
+import { Skeleton, Typography } from 'antd';
+import { useCallback, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { useInterval } from 'usehooks-ts';
+
+import { useAddress } from '@/hooks/useAddress';
+import { getLatestTransaction } from '@/service/Ethers';
+import { TransactionInfo } from '@/types/TransactionInfo';
+import { getTimeAgo } from '@/utils/time';
+
+const { Text } = Typography;
+
+const Loader = styled(Skeleton.Input)`
+  line-height: 1;
+  span {
+    width: 120px !important;
+    height: 12px !important;
+    margin-top: 6px !important;
+  }
+`;
+
+const POLLING_INTERVAL = 60 * 1000; // 1 minute
+
+/**
+ * TODO:
+ * - add cache if the same block number is requested
```

Comment:
maybe the result can be stored in storeState? and maybe not only the block number, but the last tx as well

Reply:
storeState is only really for state that needs to be persisted i.e. has the user seen their first popup when the stake

gnosis produces a new block every 5 seconds, so should be perfectly fine to pull from chain every interval more than 5s

Reply:
okidok!